src/components/TabPanel/index.tsx

```diff
@@ -50,22 +50,30 @@ export interface TabPanelProps {
     /**
      * Current index of tab
      */
-    currentTabIndex: number;
+    currentTabIndex?: number;
```

Comment:
why did you make this property not necessary?

Reply:
because we don't use it at create order component